File: android/src/main/java/io/velocitycareerlabs/reactnative/GlobalConfig.kt
/**
 * Created by Michael Avoyan on 08/08/2023.
 *
 * Copyright 2022 Velocity Career Labs inc.
 * SPDX-License-Identifier: Apache-2.0
 */

package io.velocitycareerlabs.reactnative

import io.velocitycareerlabs.api.VCLEnvironment

internal object GlobalConfig {

  var CurrentEnvironment = VCLEnvironment.PROD

  var IsDebugOn = false //BuildConfig.DEBUG

  const val LogTagPrefix = "VCL RN Android "
  val IsLoggerOn get() = CurrentEnvironment != VCLEnvironment.PROD || IsDebugOn
}

File: ios/GlobalConfig.swift
//
//  GlobalConfig.swift
//  vcl-react-native
//
//  Created by Michael Avoyan on 08/08/2023.
//
//  Copyright 2022 Velocity Career Labs inc.
//  SPDX-License-Identifier: Apache-2.0

import Foundation
import VCL

struct GlobalConfig {

    static var CurrentEnvironment = VCLEnvironment.Prod

    static var IsDebugOn = false

    static let LogTagPrefix = "VCL RN iOS "

    static var IsLoggerOn: Bool { get { CurrentEnvironment != VCLEnvironment.Prod || GlobalConfig.IsDebugOn } }
}
